.travis.yml

```diff
@@ -1,32 +1,40 @@
 language: generic
-sudo: required
+
 dist: trusty
+sudo: required
+
 services:
   - docker
+
 matrix:
-  fast_finish: true
```

Comment:
There's no reason to remove `fast_finish`. We still have multiple builds and notifying the author as soon as one of them fails is still nice to have.

Reply:
I read the fast_finish docs and it made it sound as if it was only valid with `allow_failures`.

Reply:
With fast finishing enabled, Travis CI will mark your build as finished as soon as one of two conditions are met: The only remaining jobs are allowed to fail, **or a job has already failed**.

Reply:
Oops.

Reply:
Fixed this BTW